inst/python_cmd.m

```diff
@@ -138,11 +138,8 @@
   end
 
   if (strcmp(A{1}, 'COMMAND_ERROR_PYTHON'))
-    errlineno = A{3} - LinesBeforeCmdBlock - 1;
-    errstr = sprintf(['Python raised exception, maybe at line %d of code ' ...
-                      'block (assuming popen2)\n' ...
-                      '%s'], errlineno, A{2});
-    error('OctSymPy:python_cmd:exception', errstr);
+    errlineno = A{3} - db.prelines - LinesBeforeCmdBlock;
+    error(strcat(A{2}, ' (line: ', mat2str(errlineno), ')'));
```

Comment:
mm, i prefer all in one line, like "all you need here".
and the other reason, maybe can complicate the tests...
i should test it with \n